cg/cli/balsamic.py

```diff
@@ -25,9 +26,17 @@ def balsamic(context):
 @balsamic.command()
 @click.option('-d', '--dry', is_flag=True, help='print config to console')
 @click.option('--target-bed', required=False, help='Optional')
+@click.option('--singularity', default="/home/proj/production/cancer/cases/github_issue_analysis"
```

Comment:
There is also a better path to the container: `/home/proj/production/cancer/BALSAMIC/BALSAMIC/containers/BALSAMIC.3.2.0`

Reply:
We already discussed this. But here is the issue I created on cg regarding above changes: https://github.com/Clinical-Genomics/cg/issues/453

Reply:
Thanx!

Reply:
deosn't really fit well to put such specific information as a default in the CLI. Information like this should be moved to the config